crates/web5_cli/README.md

```diff
@@ -62,6 +62,91 @@ web5 vc -h
 }
 ```
 
+### Create a `did:jwk`
```

Comment:
@aazam-gh could you rebase onto main? (The diff viewer is showing changes from #379)